main.tf

```diff
@@ -14,6 +14,13 @@ resource "aws_s3_bucket" "default" {
   acl           = "private"
   force_destroy = var.force_destroy
   tags          = module.this.tags
+  server_side_encryption_configuration {
+    rule {
+      apply_server_side_encryption_by_default {
+        sse_algorithm = "AES256"
```

Comment:
We will need this to be a feature flag so we do not break compatibility for our users.

Reply:
Interesting. Just want to learn: why not always encrypt? it does not destroy the bucket when you add this configuration and does not have noticeable cost implications.